nginx/server/sites.sls

```diff
@@ -75,6 +75,8 @@ nginx_init_{{ site.host.name }}_tls:
 
 {% endif %}
 
+{%- if grains.get('virtual_subtype', None) not in ['Docker', 'LXC'] %}
```

Comment:
Your concern was right, I fixed that by https://github.com/Martin819/salt-formula-nginx/pull/1
Once Martin merge that, we may continue the review.